scripts/manilananny.py

```diff
@@ -119,7 +120,19 @@ def list_shares(self, status=None):
             search_opts = {'all_tenants': 1}
             if status:
                 search_opts.update({'status': status})
-            return self.manilaclient.shares.list(search_opts=search_opts)
+
+            opts = copy.deepcopy(search_opts)
+            opts.update({'with_count': True})
+            _, count = self.manilaclient.shares.list(search_opts=opts)
+            count = (count / 1000) + 1
+
+            shares = []
+            offset = 0
+            for i in range(count):
```

Comment:
This needs an integer conversion

```suggestion
            for i in range(int(count)):
```